src/test/regress/expected/isolation_dump_global_wait_edges.out

```diff
@@ -84,16 +114,16 @@ step detector-dump-wait-edges:
 
 waiting_transaction_num|blocking_transaction_num|blocking_transaction_waiting
 ---------------------------------------------------------------------
-                    410|                     409|f
-                    411|                     409|f
-                    411|                     410|t
+                      9|                       8|f
```

Comment:
this has been the most easily output changing test ever :) Assigning a tx id helps, instead of updating the output I changed that